roles/ocp_on_libvirt/tasks/libvirt_host_destroy.yml

```diff
@@ -1,20 +1,31 @@
 ---
-- name: "Does node exist"
+- name: "Does node exist - {{ host['name'] }}"
+  community.libvirt.virt:
+    command: list_vms
+  register: _ool_vms_list
+
+- name: "Set node_not_exists"
+  ansible.builtin.set_fact:
+    ool_node_exists: "{{ host['name'] in _ool_vms_list.list_vms }}"
+
+- name: "Get status if exists - {{ host['name'] }}"
```

Comment:
Maybe we can merge this task and the previous one in the following way (if we also use `_ool_node_status` variable in the last task of this file):

```
- name: "Get status if exists - {{ host['name'] }}"
  vars:
    ool_node_exists: "{{ host['name'] in _ool_vms_list.list_vms }}"
  community.libvirt.virt:
    name: "{{ host['name'] }}"
    command: status
  register: _ool_node_status
  when: ool_node_exists
```

Reply:
ack, let's do that, both options are fine for me